cds-snc/forms-terraform
# feature: deploy redis and postgresql in localstack

# Summary | Résumé

Goes with https://github.com/cds-snc/platform-forms-client/pull/3421

- Moved both Redis and PostgreSQL from isolated Docker containers to Localstack Pro to align with what we have in production/staging
- Added new optional environment variables to define how Localstack Pro persistence behaves
- Reintroduce a clean option in the `localstack_services.sh` script to prevent Terraform state files from being deleted every time. This was causing deployment issues when running the script against an existing instance of Localstack where everything was already deployed.
- Updated README.md to reflect all recent changes made to the infrastructure.

# Testing

- Deploy the infrastructure using the latest information contained in the README file.
- Run and use the GC Forms web application after reviewing your `.env` file to make sure it is in sync with the latest `.env.example`.
